File: hotpot-frontend/src/types/supplier.ts
// 供应商类型定义
export interface Supplier {
  id?: number;
  name: string;
  contactPerson?: string;  // 改为contactPerson与后端一致
  phone?: string;
  address?: string;
  rating?: number;  // 添加评级字段
  status: number;
  createTime?: string;
}

File: src/main/java/com/hotpot/entity/Supplier.java
// src/main/java/com/hotpot/entity/Supplier.java
package com.hotpot.entity;

import com.baomidou.mybatisplus.annotation.*;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@TableName("supplier")
public class Supplier {
    @TableId(type = IdType.AUTO)
    private Integer id;

    private String name;
    private String contactPerson;
    private String phone;
    private String address;
    private Integer rating;
    private Integer status;

    @TableField(fill = FieldFill.INSERT)
    private LocalDateTime createTime;
}